api/src/main/java/org/motechproject/nms/api/web/contract/FrontLineWorkerUser.java

```diff
@@ -7,8 +7,8 @@
     private long currentUsageInPulses;
     private long endOfUsagePromptCounter;
```

Comment:
Sounds good. As a nitpick, @frankhuster why don't we have a Response suffix to the class name? :)